course7/homeworks/patrickh/ls/ls.cc

```diff
@@ -0,0 +1,14 @@
+#include <iostream>
+#include <boost/filesystem.hpp>
+
+int main()
+{
+	std::string path (".");
+	if (boost::filesystem::is_directory(path))
+	{
+		for (boost::filesystem::directory_iterator iterator(path); iterator!=boost::filesystem::directory_iterator(); ++iterator)
```

Comment:
It would be much easier to read if you'd use range-based-for here. Look at solutions submitted by others.